src/char/char_clif.c

```diff
@@ -885,61 +888,143 @@ int chclif_parse_charselect(int fd, struct char_session_data* sd,uint32 ipl){
 	return 1;
 }
 
+enum e_charclif_error {							//tested with 0x6e in 20150914
+	ERROR_SAMENAME_EXIST = 0x0,					//Character name already exist
+	ERROR_LIMIT_AGE = 0x1,						//You are underaged
+	//ERROR_LIMIT_CHAR_DELETE = 0x2,			
+	ERROR_INVALID_SYMBOLS = 0x2,				//Symbols in Character Names are forbidden
+	ERROR_MAKECHAR_INVALID_SLOT = 0x3,			//You are not elegible to open the Character Slot
+	ERROR_NEED_PREMIUM_SERVICE = 0xb,			//This service is only available for premium users
+	ERROR_NONPERMISSION_NAME = 0xc,				//Failed to move char slot
+	//ERROR_DELETE_LIMIT_LEVEL = 0x64,			//
+
+	ERROR_MAKECHAR_DENIED = 0xFF					//actually default message
+};
+/**
+* Acknoledge the client that the car creation failed
+* @param fd: client file descriptor
+* @param err: error that occur to inform client
+*/
+int chclif_createnewchar_ack_error(int fd, e_makechar_error err) {
+	WFIFOHEAD(fd, 3);
+	WFIFOW(fd, 0) = 0x6e;
+	switch (err) {    // lookup  makechar_error2cliff_err (map our internal error to those of the client)
+	case ERROR_INVALID_SLOT:
+	case ERROR_MAXLIMIT:
+		WFIFOB(fd, 2) = ERROR_MAKECHAR_INVALID_SLOT;
+		break; //tryme /* 0x03 = You are not elegible to open the Character Slot. */ [Ind]*/
+	case ERROR_NAME_DUPLICATE:
+	case ERROR_NAME_RESERVED:
+		WFIFOB(fd, 2) = ERROR_SAMENAME_EXIST;
+		break; 
+	case ERROR_NAME_LEN:
+	case ERROR_NAME_INVALID_CHAR:
+		WFIFOB(fd, 2) = ERROR_INVALID_SYMBOLS;
+		break;
+	case ERROR_MK_SQL:
+	case ERROR_CREATION_DISABLE:
+		WFIFOB(fd, 2) = ERROR_MAKECHAR_DENIED; break;		//255 Character creation is denied
+		break;
+	//case -3: WFIFOB(fd, 2) = 0x01; break; //wtf were -3 coming from
+	}
+	WFIFOSET(fd, 3);
+
+	return 1;
+}
+
+/**
+* Acknoledge the client that the car creation was a success
+* @param fd: client file descriptor
+* @param sd: client char_session_data
+* @param sd: new char_id created
+*/
```

Comment:
Inform the client that the character creation succeeded.
`@param` fd: client file descriptor
`@param` sd: client char_session_data
`@param` char_id: new character ID created